libs/common-services/src/test/java/uk/gov/di/ipv/core/library/service/ConfigServiceTest.java

```diff
@@ -281,51 +281,18 @@ void getCriConfigShouldReturnACriConfig() {
         @Test
         void shouldGetComponentIdForActiveConnection() {
             environmentVariables.set("ENVIRONMENT", "test");
-            final String testCredentialIssuerId = "address";
-            final String testComponentId = "testComponentId";
-            when(ssmProvider.get("/test/core/credentialIssuers/address/activeConnection"))
-                    .thenReturn("stub");
-            when(ssmProvider.get(
-                            "/test/core/credentialIssuers/address/connections/stub/componentId"))
-                    .thenReturn(testComponentId);
-            assertEquals(testComponentId, configService.getComponentId(testCredentialIssuerId));
-        }
 
-        @Test
-        void shouldLookForFeatureSetOverrideOfComponentIdOnActiveConfiguration() {
```

Comment:
This is an interesting observation - it's now a bit trickier for feature sets to override specific parts of the CRI config, but it's not been an issue so far.

I can't think of a scenario where overriding just the component ID is actually useful (vs introducing a new connection config).